### ISSUE TITLE
Rendering credentials failed with no such file error

### ISSUE THREAD

**Issue by kamatama41** [kubernetes-incubator/kube-aws#839]
I tried it following the document.
https://github.com/kubernetes-incubator/kube-aws/blob/master/Documentation/kubernetes-on-aws-render.md#tls-certificates

But I encountered the error.
```
% kube-aws version
kube-aws version v0.9.7

% kube-aws render credentials --ca-cert-path=./ca.cert.pem --ca-key-path=./ca.key.pem                                                                      [deis-staging](deis-staging)
Generating credentials...
-> Parsing existing TLS CA
-> Generating new assets
Error: open credentials/ca-key.pem: no such file or directory
```

I guess that the specified ca key is not copied into `credentials` directory so that the render command cannot read `credentials/ca-key.pem`. I'm not sure copying the ca key is correct, though.

**Comment by zmt** [comment]
Doing `kube-aws up` operations should not require the private key of the issuing ca.

**Comment by danielfm** [comment]
It does require the private key of the issuing CA if you enable TLS bootstrapping:

https://kubernetes.io/docs/admin/kubelet-tls-bootstrapping/#signing-assets

**Comment by mumoshu** [comment]
@kamatama41 Thanks for the feedback 👍
The fix is on-going in #877.
Would you mind testing it by building a kube-aws binary from the branch?
